src/lib.rs

```diff
@@ -80,13 +80,14 @@
 extern crate cortex_m;
 extern crate cortex_m_rtfm_macros;
 extern crate rtfm_core;
-extern crate static_ref;
+extern crate untagged_option;
 
 use core::u8;
 
 pub use rtfm_core::{Resource, Static, Threshold};
 pub use cortex_m::asm::{bkpt, wfi};
 pub use cortex_m_rtfm_macros::app;
+pub use untagged_option::UntaggedOption;
```

Comment:
This re-export should be `#[doc(hidden)]` as it is a not user facing implementation detail.